.github/workflows/e2e.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
        - "1.20"
        - "1.21"
        - "1.22"
+       - "1.23"
```

Comment:
Please, remove 1.20 and update the compatibility matrix in the readme, so that the next version has this new updated matrix.